core/math/math_funcs.h

```diff
@@ -197,6 +197,13 @@ _ALWAYS_INLINE_ float log(float p_x) {
 	return std::log(p_x);
 }
 
+_ALWAYS_INLINE_ double log(double p_x, double p_base) {
+	return std::log(p_x) / std::log(p_base);
+}
+_ALWAYS_INLINE_ float log(float p_x, float p_base) {
+	return std::log(p_x) / std::log(p_base);
+}
```

Comment:
Did you measure the performance of this, when invoked with the default argument?

There's a very good chance that `std::log(e)` is not optimized away -- especially since the number literal isn't specified precisely enough -- which would mean that _every_ `log(x)` call, now incur an additional logarithm + float division operation. This would make GDScript `log` slower across the board, even for functions that don't use `base` (i.e. all usages up to now).